sdk/schemaregistry/Azure.Data.SchemaRegistry/src/Generated/SchemaRestClient.cs

```diff
@@ -344,12 +364,12 @@ public async Task<ResponseWithHeaders<SchemaQueryIdByContentHeaders>> QueryIdByC
         /// <summary> Get ID for existing schema. </summary>
         /// <param name="groupName"> Schema group under which schema is registered.  Group&apos;s serialization type should match the serialization type specified in the request. </param>
         /// <param name="schemaName"> Name of schema. </param>
-        /// <param name="contentType"> Content type of the schema. </param>
+        /// <param name="contentType"> Upload file type. </param>
         /// <param name="schemaContent"> String representation (UTF-8) of the registered schema. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="groupName"/>, <paramref name="schemaName"/>, <paramref name="contentType"/> or <paramref name="schemaContent"/> is null. </exception>
+        /// <exception cref="ArgumentNullException"> <paramref name="groupName"/>, <paramref name="schemaName"/> or <paramref name="schemaContent"/> is null. </exception>
         /// <remarks> Gets the ID referencing an existing schema within the specified schema group, as matched by schema content comparison. </remarks>
-        public ResponseWithHeaders<SchemaQueryIdByContentHeaders> QueryIdByContent(string groupName, string schemaName, string contentType, Stream schemaContent, CancellationToken cancellationToken = default)
+        public ResponseWithHeaders<SchemaQueryIdByContentHeaders> QueryIdByContent(string groupName, string schemaName, Models.ContentType contentType, Stream schemaContent, CancellationToken cancellationToken = default)
```

Comment:
We probably need to come up with some conversion method to go from a SchemaFormat to a ContentType.